docs/tutorial/fmri-preprocessing-tutorial.md

```diff
@@ -101,9 +100,7 @@ Once you're happy with the surfaces, you can move onto running Freesurfer!
     * For input, select the staged preprocessed, 'acpc_aligned'-tagged anatomical (T1w & T2w) images by clicking the drop-down menu and finding the appropriate datasets.
     * Select the boxes for 'hippocampal' and 'thalamicnuclei'. Leave all other defaults the same
     ![Free Surfer Options](../img/fmri_prep_tutorial_free_surfer_options.png)
-
     ![Free Surfer Options 1](../img/fmri_prep_tutorial_free_surfer_options_1.png)
-
```

Comment:
Attempting to fix this formatting error:

<img width="696" alt="image" src="https://github.com/brainlife/docs/assets/844306/6e16d44a-d326-4de9-beb1-e8bcd60b5c35">